application/controllers/UserManagementController.php

```
@@ -895,14 +895,22 @@ public function actionImportUsers(string $importFormat = 'csv'): string
         }
         $created = [];
         $updated = [];
+        $hasDuplicateEmails = false;
+        $canOverwriteDuplicateEmail = false;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$canOverwriteDuplicateEmail'. (undefined)

(UnusedLocalVariable)